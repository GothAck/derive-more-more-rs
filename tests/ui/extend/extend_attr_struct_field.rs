use derive_more_more::Extend;

#[derive(Default, Extend)]
#[extend(usize)]
struct NamedMultiAttrStructField {
    #[extend(usize)]
    inner: Vec<usize>,
    test: bool,
}

#[derive(Default, Extend)]
#[extend(usize)]
struct NamedSingleAttrStructField {
    #[extend(usize)]
    inner: Vec<usize>,
}

#[derive(Default, Extend)]
#[extend(usize)]
struct UnnamedMultiAttrStructField(#[extend(usize)] Vec<usize>, bool);

#[derive(Default, Extend)]
#[extend(usize)]
struct UnnamedSingleAttrStructField(#[extend(usize)] Vec<usize>);

fn main() {}
