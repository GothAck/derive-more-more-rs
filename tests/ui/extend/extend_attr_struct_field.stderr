error: Expected `extend` attribute on struct or one field
 --> tests/ui/extend/extend_attr_struct_field.rs:4:1
  |
4 | #[extend(usize)]
  | ^

error: Expected `extend` attribute on struct or one field
  --> tests/ui/extend/extend_attr_struct_field.rs:12:1
   |
12 | #[extend(usize)]
   | ^

error: Expected `extend` attribute on struct or one field
  --> tests/ui/extend/extend_attr_struct_field.rs:19:1
   |
19 | #[extend(usize)]
   | ^

error: Expected `extend` attribute on struct or one field
  --> tests/ui/extend/extend_attr_struct_field.rs:23:1
   |
23 | #[extend(usize)]
   | ^
